patches/build-android-pylib-gtest-gtest_test_instance.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/android/pylib/gtest/gtest_test_instance.py b/build/android/pylib/gtest/gtest_test_instance.py
+index e4b76b321e4e1e4b630c37e779221bb9f0068c21..15035d64cd295afc198cc85afc6f64e86a71ffaa 100644
+--- a/build/android/pylib/gtest/gtest_test_instance.py
++++ b/build/android/pylib/gtest/gtest_test_instance.py
+@@ -31,6 +31,7 @@ BROWSER_TEST_SUITES = [
+     'content_browsertests',
+     'weblayer_browsertests',
+ ]
++from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
```

Comment:
in most cases you should put this line either before `if __name__ == '__main__':` or at the end of the file. 

In this exact case there's no `__main__` thingy, and the file only declares some types without their instantiation, so it is safe and more patch-friendly (to never touch this line again) to put this line at the end of the file.

https://github.com/brave/brave-browser/wiki/Patching-Chromium#patching-py-files

Reply:
thanks, fixed at 
https://github.com/brave/brave-core/compare/8894bac99b2328d0f02b29cd556e9d0f4987c8d6..4baf2f090143e2781a8f8dcadf70a14ae0490dfb